test.js

```diff
@@ -49,6 +49,35 @@ function describeTestsWithOptions(options, postText) {
       const res = utils.getAttrs(replaceDelimiters(src, options), 0, options);
       assert.deepEqual(res, expected);
     });
+
+    it(replaceDelimiters('should parse attributes whose are ignored the key chars(\\t,\\n,\\f,\\s,/,>,",\',=) eg: {gt>=true slash/=trace i\\td "q\\fnu e\'r\\ny"=}', options), () => {
```

Comment:
Why should one ignore these chars? What is the intended use case?

Reply:
When I run `npm test` and saw that were prompt:

File         | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
-------------|---------|----------|---------|---------|------------------------------------
All files    |    93.1 |    87.15 |   88.63 |   93.98 |
 index.js    |   87.34 |    80.64 |   78.57 |   90.32 | 117-124,138
 patterns.js |     100 |    95.23 |     100 |     100 | 327
 utils.js    |   92.12 |    89.58 |   83.33 |   91.86 | 81,145,198,237,241,253,270,274-277

This test case is covered the utils.js:L81 and reminded by the notes on L8-9:

https://github.com/arve0/markdown-it-attrs/blob/80a8e5278fb336725bcdde14b7c93ec8fbaafc04/utils.js#L8-L9